### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by luizcmarin**
readme.md, links in documentation does not exist, have been removed:

```
- [`config/common.php`](https://github.com/yiisoft/mailer-symfony/blob/master/config/common.php)
- [`config/params.php`](https://github.com/yiisoft/mailer-symfony/blob/master/config/params.php)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This package is an adapter for [yiisoft/mailer](https://github.com/yiisoft/maile
 
 ## Installation
 
-The package could be installed with [Composer](https://getcomposer.org):
+The package could be installed with composer:
```
Comment (vjik) [inline README.md:27]: Keep link to composer

File: README.md [hunk 2]
```diff
@@ -84,19 +84,21 @@ For more information about signing and encrypting messages, see the correspondin
...
 
-For use in the [Yii framework](https://www.yiiframework.com/), see the configuration files:
-  - [`config/di.php`](https://github.com/yiisoft/mailer-symfony/blob/master/config/di.php)
-  - [`config/params.php`](https://github.com/yiisoft/mailer-symfony/blob/master/config/params.php)
-
 ## Documentation
 
-- [Yii guide to mailing](https://github.com/yiisoft/docs/blob/master/guide/en/tutorial/mailing.md)
-- [Symfony Mailer documentation](https://symfony.com/doc/current/mailer.html)
+- See [Yii guide to mailing](https://github.com/yiisoft/docs/blob/master/guide/en/tutorial/mailing.md)
```
Comment (vjik) [inline README.md:89]: Keep as is.

File: docs/internals.md [hunk 3]
```diff
@@ -37,7 +37,9 @@ use either newest or any specific version of PHP:
 ## Dependencies
 
 Use [ComposerRequireChecker](https://github.com/maglnet/ComposerRequireChecker) to detect transitive
-[Composer](https://getcomposer.org/) dependencies. To run the checker, execute the following command:
+[Composer](https://getcomposer.org/) dependencies.
```
Comment (vjik) [inline docs/internals.md:40]: Keep as is.

**Review by vjik — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -84,19 +84,21 @@ For more information about signing and encrypting messages, see the correspondin
...
 
-For use in the [Yii framework](https://www.yiiframework.com/), see the configuration files:
-  - [`config/di.php`](https://github.com/yiisoft/mailer-symfony/blob/master/config/di.php)
-  - [`config/params.php`](https://github.com/yiisoft/mailer-symfony/blob/master/config/params.php)
-
 ## Documentation
 
 - [Yii guide to mailing](https://github.com/yiisoft/docs/blob/master/guide/en/tutorial/mailing.md)
 - [Symfony Mailer documentation](https://symfony.com/doc/current/mailer.html)
+- For use in the [Yii framework](https://www.yiiframework.com/), see the configuration files:
```
Comment (vjik) [inline README.md:91]: Currently place of this block is better.

**Review by vjik — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (luizcmarin) [reply README.md:91]: I didn't understand. What should I do?

Comment (vjik) [reply README.md:91]: Block "For use in the yii framework" keep as is.

File: docs/internals.md [hunk 5]
```diff
@@ -39,6 +39,8 @@ use either newest or any specific version of PHP:
 Use [ComposerRequireChecker](https://github.com/maglnet/ComposerRequireChecker) to detect transitive
 [Composer](https://getcomposer.org/) dependencies. To run the checker, execute the following command:
 
+To run the checker, execute the following command:
```
Comment (vjik) [inline docs/internals.md:42]: It's duplicate.

**Review by vjik — CHANGES_REQUESTED** [review]

File: docs/internals.md (hunk 5 above)
Comment (luizcmarin) [reply docs/internals.md]: ok. done

**Review by vjik — APPROVED** [review]

**Pull request merged**